### PR TITLE
Bump vite

### PR DESCRIPTION
**Pull request by jsandfordhughescoop**
This bumps Vite to v6

### CODE REVIEW

**Comment by jsandfordhughescoop** [comment]
@fabio-ivona - I believe this should be a quick merge as Vite v6 does not have any breaking changes.

**Comment by fabio-ivona** [comment]
thanks!

**Comment by Jas-n** [comment]
May I suggest moving this change to a v3 release, similarly to how vite 5 was v2?

In a minor version bump it causes a dependency error, which may not always be a quick fix.

image

👍

File: package.json [hunk 1]
```diff
@@ -46,11 +46,11 @@
         "esbuild": "0.23.0",
         "eslint": "^8.56.0",
         "typescript": "^5.3.3",
-        "vite": "^5.0.0",
+        "vite": "^6.0.2",
         "vitest": "^2.0.1"
     },
     "peerDependencies": {
-        "vite": "^5.0.0"
+        "vite": "^6.0.2"
```
Comment (Jas-n) [inline package.json:53]: Keep support for Vite 5:

```json
"vite": "^5.0.0 || ^6.0.0"
```

Comment (fabio-ivona) [reply package.json:53]: done

**Pull request merged**